lib/ezcater_rubocop/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module EzcaterRubocop
-  VERSION = "3.0.2"
+  VERSION = "3.0.3"
```

Comment:
Per [semver](https://semver.org/), should this be a minor bump given that it's adding a new feature?

Reply:
I'd be okay with a major version bump.